css/styles.css

```diff
@@ -6,45 +6,56 @@
   box-sizing: border-box;
 }
 
+body {
+  background-color: lightslategrey;
+  margin: 0;
+  padding: 0;
+}
+
 .header {
-  border-radius: 0px 0px 10px 10px;
+  border-radius: 10px 10px 10px 10px;
   display: flex;
   justify-content: center;
-  background-color: aquamarine;
-  box-shadow: 5px 5px 5px grey;
+  background-color: white;
+  box-shadow: 4px 4px 4px 2px;
```

Comment:
you forgot to add a color to your box shadow, unless you want a really harsh black drop shadow. the 4 numbers are not to be understood as with padding or margin btw: the first two repesent the offset of the shadow, the third one the blur factor and the last one the spread factor.

Reply:
I might look Into that I mean black is not bad but I could use a grey shadow instead. 

so basically I wanted a shadow on the box showing only on the button a bit more then on the right box side so that is why I used 4 px and 2 px.